### ISSUE TITLE
Refactor image pipes across the modules

### ISSUE THREAD

**Issue by bethac07** [CellProfiler/CellProfiler#3077]
Sphinx gets cranky when image pipes have the same name, so we need to give images unique names- ie `|Help_BuildPipe_image0|` rather than just `|image0|`

My recommendation is to do `|ModuleNameOrAcronym_FunctionCall_imageNum|` .  I've already done the top level ones as just `|ModuleNameOrAcronym_imageNum|`

**Comment by AnneCarpenter** [comment]
I bet @N3llz could do this if you explain how.

**Comment by mcquin** [comment]
This may be less necessary after #3137. Let's investigate once that is merged.

**Comment by bethac07** [comment]
This seems now nice but unnecessary.  Closing as not a priority, though some patient soul is welcome to take it on in the future.
